packages/InputGroup/style.js

```diff
@@ -0,0 +1,69 @@
+import styled from 'styled-components'
+
+export const LabelStyle = styled.label(() => ({
+  span: {
+    fontSize: '14px',
+    color: '#2A2C2E',
+    display: 'block',
+  },
+}))
+
+export const InputGroupStyle = styled.div(({ hasValue }) => ({
+  display: 'inline-flex',
+  flexWrap: 'nowrap',
+  height: '52px',
+  width: '100%',
+  borderRadius: '5px',
+  '&:hover': {
+    boxShadow: '0 0 0 1px #4B286D',
+    input: {
+      borderColor: '#4B286D',
+    },
+  },
+  input: {
+    height: '100%',
+    borderLeft: '1px solid #D8D8D8',
+    borderTop: '1px solid #D8D8D8',
+    borderBottom: '1px solid #D8D8D8',
+    borderRight: 'none',
+    borderRadius: '4px 0 0 4px',
+    color: '#2A2C2E',
+    flexGrow: 1,
+    marginRight: hasValue ? '-52px' : '0',
+    textIndent: '15px',
+    '&::-ms-clear': {
+      display: 'none',
+    },
+    '&::placeholder': {
+      color: '#71757B',
+    },
+    '&:active': {
+      borderColor: '#4B286D',
+    },
+    '&:focus': {
+      borderColor: '#4B286D',
+    },
+  },
+  button: {
+    height: '100%',
+    width: '52px',
+    cursor: 'pointer',
+    '&:last-child': {
+      backgroundColor: '#462C6A',
+      borderLeft: 'none',
+      borderTop: '1px solid #462C6A',
+      borderRight: '1px solid #462C6A',
+      borderBottom: '1px solid #462C6A',
+      borderRadius: '0 4px 4px 0',
+      i: {
+        color: '#FFFFFF',
+      },
+    },
+  },
+  'input:focus, button:last-child:focus': {
+    outlineWidth: '4px',
```

Comment:
Instead of outline, can we use box-shadow on focus. The outline looks very square and abrupt and doesn't match the visual design. You can achieve rounded corners using box-shadow.

Reply:
I did consider using box shadow. However, there is 1 px space between outline and input which I do not believe it can be achieved by box-shadow

Reply:
Hmmm, it looks a bit abrupt with the square outline. I think you could do this with a :before psuedo class and using box shadow

Reply:
> Hmmm, it looks a bit abrupt with the square outline. I think you could do this with a :before psuedo class and using box shadow

Sure, I can definitely give it a try. thanks

Reply:
Thanks!

Reply:
Hi Mike, FYI I think we can only add psuedo element inside of container elements. Input element does not support it. I talked to Laurel and using rectangle outline is acceptable. 

`Pseudo-elements can only be defined (or better said are only supported) on container elements. Because the way they are rendered is within the container itself as a child element. input can not contain other elements hence they're not supported.`

Reply:
ok, no worries. Lets keep it as it is, but can you add: `'-moz-outline-radius': '8px',` so at least it will have rounded corners in Firefox which supports it. Thanks

Reply:
> ok, no worries. Lets keep it as it is, but can you add: `'-moz-outline-radius': '8px',` so at least it will have rounded corners in Firefox which supports it. Thanks

Ok, I will add it